### DISCUSSION TITLE
Feature Requests

### DISCUSSION THREAD

**Discussion by rohanjsh** [rohanjsh/typeset#37] [Ideas]
Would love to hear from everyone!

**Comment by rohanjsh** [comment]
- [ ] Support for Color property https://www.reddit.com/r/FlutterDev/s/uAmrPEsFTX - #41

**Reply by rohanjsh** [reply]
Multiple request for this feature. What do you think, is it achievable by <Size, ColorHexCode> syntax? @FirdousNath 
For example:
`TypeSet('Hello *World<20, #FFFFFF>*')` for world to be Bold + sized 20 + white color

**Reply by FirdousNath** [reply]
GTG, extra split by coma will be needed after font regex

**Comment by arthurbcd** [comment]
`Text` extension?

```dart
Text('I have *superpowers*!').asTypeset()
```

This would imply that `typeset` has the same API and that we can easily migrate.

Maybe `.toTypeset()` or simply `.typeset()`. 

Love this package, btw. 😄

**Reply by rohanjsh** [reply]
Hey @arthurbcd,
Thank you for your suggestion. Umm, let me think about it.

Currently though, we have a string extension in place for typeset;
Eg: `'I have *superpowers*!'.typeset()`
Ref: https://github.com/rohanjsh/typeset/blob/dev/lib/src/view/typeset_ext.dart
Will this work for you? 🧐
